service/history/api/get_history_util.go

```diff
@@ -74,12 +73,7 @@ func GetRawHistory(
 		return nil, nil, err
 	}
 
-	for _, data := range resp.HistoryEventBlobs {
-		rawHistory = append(rawHistory, &commonpb.DataBlob{
-			EncodingType: data.EncodingType,
-			Data:         data.Data,
-		})
-	}
+	rawHistory := resp.HistoryEventBlobs
```

Comment:
nit: can we make this and the file above consistent? The one in history service is meant to be the copy of the one in frontend.

Reply:
done!